.circleci/config.yml

```diff
@@ -28,7 +28,16 @@ references:
   build_docker_image: &build_docker_image
     run:
       name: Build laa-fee-calculator docker image
-      command: docker build --build-arg DJANGO_SECRET_KEY=${DJANGO_SECRET_KEY} -t app .
+      command: |
+        export BUILD_DATE=$(date -Is) >> $BASH_ENV
+        source $BASH_ENV
+        docker build \
+          --build-arg DJANGO_SECRET_KEY=${DJANGO_SECRET_KEY} \
+          --build-arg VERSION_NUMBER=${CIRCLE_BUILD_NUM} \
+          --build-arg COMMIT_ID=${CIRCLE_SHA1} \
+          --build-arg BUILD_DATE=${BUILD_DATE} \
+          --build-arg BUILD_TAG=${CIRCLE_BRANCH} \
```

Comment:
Personally I would imagine build tag to be `branch_name.first_seven_chars_of_commit_id` but that isn't really adding any extra information to the `ping.json` response as a whole so whatever.

Reply:
Happy to discuss... my thinking was that COMMIT_ID gives us the SHA, so having the branch name would let us see if the environment has **master** or **branch-name** deployed to it, without having to cross reference back to Github/CircleCI